### ISSUE TITLE
Unable to Build NVDA: Error While Compiling Phoneme Data

### ISSUE THREAD

**Issue by TylerK93** [nvaccess/nvda#6041]
When I try to compile NVDA from either the master or next branches, the build goes well all the way to the point where the ESpeak phoneme data is trying to be compiled. When that step is reached, the build fails with this error:

scons: *** [include\espeak\espeak-data\intonations] %1 is not a valid Win32 application

**Comment by derekriemer** [comment]
In my experience, this is due to one of two things, probaly 1.


1. Are you running python 64 bit? NVDA expects python 32 bit. It will 
crash with this error if you have the wrong bitness.


2. Did you do a git submodule update in the root before this after 
switching ranches?


On 6/6/2016 10:19 AM, Tyler Kavanaugh wrote:
>
> When I try to compile NVDA from either the master or next branches, 
> the build goes well all the way to the point where the ESpeak phoneme 
> data is trying to be compiled. When that step is reached, the build 
> fails with this error:
>
> scons: *** [include\espeak\espeak-data\intonations] %1 is not a valid 
> Win32 application
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/6041>, or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivaM9Pba9M6jf9xQFScgeBS-j3tzHks5qJEitgaJpZM4IvEY1>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by TylerK93** [comment]
Oh...duh. It seems I have Python 2.7.11 64-bit. I probably should have installed 32-bit via Chocolatey...is there no way to get the build system to accept 64-bit Python? Or is this a limitation imposed by the eSpeak build system?

**Comment by derekriemer** [comment]
https://github.com/nvaccess/nvda and go to dependencies.

There, it says 32 bit.

You might be able to get away with some weird virtualenv setup or something


On 6/6/2016 11:47 AM, Tyler Kavanaugh wrote:
>
> Oh...duh. It seems I have Python 2.7.11 64-bit. I probably should have 
> installed 32-bit via Chocolatey...is there no way to get the build 
> system to accept 64-bit Python? Or is this a limitation imposed by the 
> eSpeak build system?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/6041#issuecomment-224033913>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivUvtpo9IS9lkmPVklP6zQVf5TrNvks5qJF0_gaJpZM4IvEY1>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

**Comment by jcsteh** [comment]
NVDA is a 32 bit app so that it can handle both 32 bit and 64 bit apps and systems. Thus, it depends on 32 bit Python.
